virttest/qemu_vm.py

```diff
@@ -1683,13 +1688,20 @@ def add_secure_guest_descriptor(params):
         devices.insert(StrDev('vmname', cmdline=add_name(name)))
 
         qemu_sandbox = params.get("qemu_sandbox")
+        sandbox_params = Params()
+        prefix = 'qemu_sandbox_'
+        for key in params:
+            if key.startswith(prefix):
+                new_key = key.rsplit(prefix)[1]
+                sandbox_params[new_key] = params[key]
```

Comment:
qemu only has 1 sandbox, I don't think we need to iterate through the entire params. For me, 2 options:
1. `qemu_sandbox_options = xxx=111 yyy=222`, ','.join(params.get_list("qemu_sandbox_options")), or directly use `qemu_sandbox_options = ,xxx=111,yyy=222`
2. Still controlling all options as individual parameters, but using `params.copy_from_keys()`, option names can be saved in a parameter.